cli/src/commands/upload.command.ts

```diff
@@ -48,28 +49,18 @@ class Asset {
     let sidecarData: Blob | undefined = undefined;
     try {
       await access(sideCarPath, constants.R_OK);
-      sidecarData = new File([await fs.openAsBlob(sideCarPath)], basename(sideCarPath));
+      sidecarData = await fs.openAsBlob(sideCarPath);
```

Comment:
Why doesn't this need to be a new file, but the asset data does?

Reply:
After testing I reverted this.